frontend/cypress/README.md

```diff
@@ -1,13 +1,23 @@
-# Cypress Integration Tests
+# Cypress Visual Testing with BDD framework
```

Comment:
This just a nit: "visual testing" is its own thing and means something specific outside of testing with cypress. Calling this something else will help avoid the confusion.

Reply:
you are right, I will review this once more as I will progress on documentation